packages-common/pluggable-widgets-tools/tests/commands.js

```diff
@@ -0,0 +1,204 @@
+const mapLimit = require("async/mapLimit");
+const { Mutex } = require("async-mutex");
+const { exec } = require("child_process");
+const { copy, readJson, writeJson } = require("fs-extra");
+const { join } = require("path");
+const { mkdir, rm, tempdir } = require("shelljs");
+const kill = require("tree-kill");
+const { promisify } = require("util");
+const { run: runYeoman } = require("yeoman-test");
+
+const LIMIT_TESTS = !!process.env.LIMIT_TESTS;
+
+const CONFIGS = [
+    ["web", "full", "ts", "8.0"],
+    ["native", "full", "js", "8.1"],
```

Comment:
@diego-antonelli @IIsaku how about dropping a few configs out of this list? Say, 8.1 and 8.3?

Reply:
Im fine with that :)

Reply:
Removed them.